src/analysis/retail/evoker/augmentation/modules/abilities/ShiftingSands.tsx

```diff
@@ -134,8 +161,13 @@ class ShiftingSands extends Analyzer {
         ),
       };
     }
+    return presciencePerformance;
+  }
 
+  private getRolePerformance(application: ShiftingSandsApplications) {
+    const className = application.combatant.spec?.className.replace(/\s/g, '') ?? '';
```

Comment:
not a huge fan of generating a css class name on the fly like this, but it does work sooooo :thinking: